docs/how-tos/configure.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Build and configure smart machine"
-linkTitle: "Configure a smart machines"
+title: "Build and configure a smart machine"
```

Comment:
[opt] 
```suggestion
title: "Configure a smart machine"
```

Reply:
Maybe create and configure. I think only we think of the configuration aspects as how you build a representation of a machine within Viam. 
```suggestion
title: "Create and configure a smart machine"
```